pkg/porter/explain.go

```diff
@@ -253,7 +257,9 @@ func generatePrintable(bun bundle.Bundle) (*PrintableBundle, error) {
 		po.ApplyTo = generateApplyToString(v.ApplyTo)
 		po.Description = v.Description
 
-		outputs = append(outputs, po)
+		if po.ApplyTo == "All Actions" || po.ApplyTo == actionOption || actionOption == "" {
```

Comment:
Thanks for catching that outputs also should be filtered! 👍 

The `Output` type also has an AppliesTo function that we can use

```suggestion
		if v.AppliesTo(action){
```